apps/prairielearn/package.json

```diff
@@ -53,6 +53,7 @@
     "@prairielearn/tsconfig": "workspace:^",
     "@prairielearn/workspace-utils": "workspace:^",
     "@pyroscope/nodejs": "^0.4.1",
+    "@smithy/types": "^3.7.2",
```

Comment:
I believe this should be a dev dependency.

Reply:
Oops -- thanks.